File: camerashop_fe/src/types/ResponseObject.ts
// types/ResponseObject.ts

export interface ResponseObject {
    status: string;  // Trạng thái của phản hồi, ví dụ 'ok' hoặc 'error'
    message: string; // Thông điệp về kết quả
    data?: any;      // Dữ liệu có thể trả về, có thể là bất kỳ kiểu dữ liệu nào
}

File: document/backup/camerashop_be1/src/main/java/com/example/camerashop_be/entity/ResponseObject.java
package com.example.camerashop_be.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ResponseObject implements Serializable {
	private String status;
	private String message;
	private Object data;
}
